Client/mods/deathmatch/logic/CResource.cpp

```diff
@@ -267,14 +267,15 @@ void CResource::Load()
     }
 
     // Load the no cache scripts first
-    for (std::list<SNoClientCacheScript>::iterator iter = m_NoClientCacheScriptList.begin(); iter != m_NoClientCacheScriptList.end(); ++iter)
-    {
+    for (auto& item : m_NoClientCacheScriptList) {
```

Comment:
This is subjective, but you should only use auto when the type is visible [on the right of an assignment].

```suggestion
    for (SNoClientCacheScript& item : m_NoClientCacheScriptList) {
```

Reply:
The type of the item is not important in this context, that's why I use auto.